build.gradle

```diff
@@ -2,18 +2,19 @@
 
 buildscript {
     repositories {
-        jcenter()
         mavenCentral()
+        google()
+        jcenter()
```

Comment:
sometimes when doing a clean it will cause a problem **Could not find play-services-basement.aar**  [url](https://stackoverflow.com/questions/50563407/could-not-find-play-services-basement-aar)